components/molecule/dropdownList/src/index.js

```diff
@@ -107,10 +107,8 @@ const MoleculeDropdownList = forwardRef(
           .map((child, index) => (
             <ExtendedChildren
               key={index}
-              index={index}
               value={value}
               onSelect={onSelect}
-              onKeyDown={onKeyDown}
```

Comment:
This causes the bug mentioned

Reply:
does that mean (removing passing down that `prop`) that the user can no longer navigate through the options with the keyboard?

Reply:
no, he can. Is fired by its wrapper via event bubbling

Reply:
gotcha! thanks